nss-ivory-trolls/BangazonAPI
# Ac customer controller

# Description
Feature adds CRUD functionality to Customer Controller. Added automated testing to Customer Controller. Updated ReadMe file

## Type of change

# Testing Instructions
1. git checkout ac-customer-controller
1. dotnet run
1. open PostMan

Ensure the following HTTP requests are functional:
- Get (All)
- Get (Id)
- Post
- Put

1. Navigate to TestBangazonAPI project. Ensure that all tests in CustomerCRU file pass. 

# Checklist:
